dnf/base.py

```diff
@@ -890,13 +891,15 @@ def _trans_error_summary(self, errstring):
         """
         summary = ''
         # do disk space report first
-        p = re.compile(r'needs (\d+)MB on the (\S+) filesystem')
+        p = re.compile(r'needs (\d+)(K|M)B on the (\S+) filesystem')
```

Comment:
Are we sure we can only get "K" or "M" here?

Reply:
Yes I checked RPM code